config/suppressions.xml

```diff
@@ -8,6 +8,9 @@
     <!-- can't split long messages between lines -->
     <suppress checks="RegexpSingleline" files="google_checks\.xml" lines="42,83"/>
 
+    <!-- Ignore examples of invalid code -->
+    <suppress checks="RegexpMultiline" files="config_whitespace\.xml"/>
```

Comment:
@romani Should this have specific line numbers like google_checks does?

Reply:
we need to define id for module instance https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_checks.xml#L75 and suppress by certain id.
https://checkstyle.org/config_filters.html#SuppressionFilter
```
<suppress id="unnecessaryConsecutiveLines"
    files="config_whitespace\.xml"
    lines="350-500"/>
```

but ..... can we use some fancy escaping symbols instead of simple "\n" ?

Reply:
done. Good news is that `&#10;` is used only on HTML side, in the java code these lines are not empty, they are prepend by asterisks.